docs/tutorial/tutorial.md

```diff
@@ -0,0 +1,126 @@
+```
+
+### Deploy and relate database
+
+#### Database relation
+
+The following commands deploys the mysql-k8s charm and relates wordpress-k8s charm through its mysql
+interface.
+
+```
+juju deploy mysql-k8s \
+    --config mysql-interface-user=wordpress-tutorial-user \
+    --config mysql-interface-database=wordpress-tutorial-database
+juju relate wordpress-k8s mysql-k8s
+```
+
+#### Database configuration
+
+Note: This section is required only if database relation is not used. If you want to deploy your
+wordpress-k8s charm through configuration, you may continue to follow the steps in this section.
+
+The commands below create the MySQL deployment in microk8s which we can then use to configure the
+database for wordpress-k8s. Note that the environment variables are for demonstration purposes only
+and should not be used for production environments.
+
+```
+microk8s kubectl run mysql -n wordpress-tutorial --image=mysql:latest \
+--env="MYSQL_ROOT_PASSWORD=<strong-password>" \
+--env=”MYSQL_DATABASE=wordpress-tutorial-database” \
+--env=”MYSQL_USER=wordpress-tutorial-user” \
+--env=”MYSQL_PASSWORD=<strong-password>
+WORDPRESS_IP=microk8s kubectl get pod mysql --template '{{.status.podIP}}'
+juju config wordpress-k8s \
+db_host=$WORDPRESS_IP \
+db_name=wordpress-tutorial-database \
+db_user=wordpress-tutorial-user \
+db_password=<strong-password>
+```
+
+Use the following command to watch your deployment progress through different stages of deployment.
+
+```
+juju status --color --watch 2s
+```
+
+### Get admin credentials
+
+After the database has been configured in the
+[Deploy and relate database section](#deploy-and-relate-database), you can now access the WordPress
+application by accessing the IP of a wordpress-k8s unit. To start managing WordPress as an
+administrator, you need to get the credentials for the admin account.
+
+By running the `get-initial-password` action on a wordpress-k8s unit, juju will read and fetch the
+admin credentials setup for you. You can use the following command below.
+
+```
+juju run-action wordpress-k8s/0 get-initial-password --wait
+```
+
+The result should look something similar to the contents below:
+
+```
+unit-wordpress-k8s-0:
+  UnitId: wordpress-k8s/0
+  id: "6"
+  results:
+    password: <password> # should look something like: 7ioJgp2CS6iK7MrM9I-wCm48i25HaJLzeHT7MpEwdyk
+  status: completed
+  timing:
+    completed: 2023-02-24 02:46:27 +0000 UTC
+    enqueued: 2023-02-24 02:46:25 +0000 UTC
+    started: 2023-02-24 02:46:26 +0000 UTC
+```
+
+You can now access your WordPress application at `http://<UNIT_IP>/wp-login.php` and login with
+username admin and password the action above.
+
+### Cleaning up the environment
+
+Congratulations! You have successfully finished the wordpress-k8s tutorial. You can now remove the
+model environment that you’ve created using the following command.
+
+```
+juju destroy model wordpress-tutorial -y --release-storage
+```
+
+### Next steps
```

Comment:
I don't think we'll need this section

Reply:
Seems like none of the tutorials have such steps, removed :)